bench.sh

```diff
@@ -536,7 +553,7 @@ else
   # XXX cabal issue "cabal v2-exec which" cannot find benchmark/test executables
   #WHICH_COMMAND="cabal v2-exec which"
   WHICH_COMMAND=cabal_which
-  BUILD_CHART_EXE="cabal v2-build --flags dev chart"
+  BUILD_CHART_EXE="cabal v2-build --builddir=$CABAL_BUILD_DIR --flags dev chart"
```

Comment:
Is this still needed?

Reply:
You're commenting on outdated code sir. Outdated I say.